CHANGELOG.md

```diff
@@ -5,6 +5,11 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ❤️ **Donate:** Enjoying MagicMirror²? [Please consider a donation!](https://magicmirror.builders/donate) With your help we can continue to improve the MagicMirror² core.
 
+## [2.11.0] - 2020-01-31
```

Comment:
The CHANGELOG.md in de develop branch already contains entries for the 2.11 release. Please rebase your PR on the develop branch.